osf/migrations/__init__.py

```diff
@@ -125,3 +125,30 @@ def update_permission_groups(sender, verbosity=0, **kwargs):
     if getattr(sender, 'label', None) == 'osf':
         update_admin_permissions(verbosity)
         update_provider_auth_groups(verbosity)
+
+
```

Comment:
Drive by: Will this overwrite any existing preprint moderator notification settings?

Reply:
No, there's a check in `add_user_to_subscription` that makes it a noop if the person is already subscribed.

Reply:
If they did, then they should be in the none relationship of the notification